.github/workflows/nightly.yaml

```diff
@@ -96,6 +106,45 @@ jobs:
             TAG=${{ steps.meta.outputs.version }}
             COMMIT=${{ github.sha }}
             REPO_INFO=https://github.com/${{ github.repository }}.git
+  rh-scan-registry:
+    runs-on: ubuntu-latest
+    needs: build-push-images
+    steps: 
+      - name: Login to RedHat Scan Registry
+        uses: docker/login-action@v2
+        with:
+          registry: scan.connect.redhat.com
+          username: ${{ secrets.RH_USERNAME }}
+          password: ${{ secrets.RH_TOKEN }}
+      - name: Docker meta (redhat scan registry)
+        id: meta_redhat_scan_registry
+        uses: docker/metadata-action@v4.0.1
+        with:
+          images: ${{ env.RH_SCAN_REGISTRY }}/${{ env.RH_SCAN_REGISTRY_IMAGE_NAME }}
```

Comment:
If these are only needed in this job then we can set them only for this jobl via: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idenv